Traceback (most recent call last):
  File "run_pproc.py", line 97, in <module>
  File "run_pproc.py", line 89, in main
  File "run_pproc.py", line 65, in run_preprocess
  File "run_pproc.py", line 31, in dispatch_process
  File "/home/ittay.al/miniconda3/envs/tf1/lib/python3.7/multiprocessing/process.py", line 112, in start
  File "/home/ittay.al/miniconda3/envs/tf1/lib/python3.7/multiprocessing/context.py", line 223, in _Popen
  File "/home/ittay.al/miniconda3/envs/tf1/lib/python3.7/multiprocessing/context.py", line 277, in _Popen
  File "/home/ittay.al/miniconda3/envs/tf1/lib/python3.7/multiprocessing/popen_fork.py", line 20, in __init__
  File "/home/ittay.al/miniconda3/envs/tf1/lib/python3.7/multiprocessing/popen_fork.py", line 70, in _launch
OSError: [Errno 12] Cannot allocate memory
